### ISSUE TITLE
0.7.4 Release

### ISSUE THREAD

**Issue by tgriesser** [tgriesser/knex#710]
### tl;dr 0.7.4 is released with some recent patches

As cutting a new release with the current state of master would have required bumping the minor version, and there's a few things that should get done before that is ready, I just cut a release with some cherry-picked patches starting off 0.7.3's commit.

If you see any bugs or feel like anything minor missed the cut, feel free to open a ticket here or open any PR's against the 0.7 branch and we'll make sure we bump the minor and then merge them into master.

In the future I'd like to get a better branching / versioning strategy so we can work on big things like pool rewrites without sacrificing things like minor bugfixes which can't be accomplished if things are just always merged into master.

/cc @leore thanks for your patience on the limit issue :)

**Comment by leore** [comment]
@tgriesser thanks for this. Gonna use this version today and let you know if I see any bugs

**Comment by leore** [comment]
@tgriesser everything working. thanks!
